config/locales/views/admin/en.log_events.yml

```diff
@@ -27,6 +27,9 @@ en:
           problem_queue: Order added to problem queue %{cause}
           price_change: Order's price was manually changed
           resolve_from_problem_queue: Problem order resolved
+        facility:
+          activate: Relays turn on
```

Comment:
I'm trying to think of a better phrase. Not sure what's best

- "Turned on all relays"
- "Global relay activated"
- "All relays activated"

Reply:
And if we change this, we should update the deactivate to match.